src/platform-integration/ios/platform-views.md

```diff
@@ -112,6 +112,10 @@ class FLNativeViewFactory: NSObject, FlutterPlatformViewFactory {
             arguments: args,
             binaryMessenger: messenger)
     }
+
+    public func createArgsCodec() -> FlutterMessageCodec & NSObjectProtocol {
```

Comment:
Ditto with my comment on the Objective-C version.

Reply:
Dear @cyanglaz , I added comment as your request.